### PR TITLE
Version 1.3.0

### PR DESCRIPTION
**Pull request by Scotchester**
Just a version bump and changelog update.
## Review
- @jimmynotjim

### CODE REVIEW

**Comment by Scotchester** [comment]
Bump @jimmynotjim

**Comment by jimmynotjim** [comment]
:+1:

**Comment by jimmynotjim** [comment]
Bump back @Scotchester

**Comment by Scotchester** [comment]
O HAI

**Comment by jimmynotjim** [comment]
@Scotchester Make sure to tag and release

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +23,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Fixed
 
-- typo in position delcaration
+- typo in position declaration
```
Comment (jimmynotjim) [inline CHANGELOG.md:26]: Lol, typo about a typo, so meta

**Pull request merged**